recipes/recipes/emscripten_emscripten-32/activate.sh

```diff
@@ -1,13 +1,24 @@
 
 if [ -z ${CONDA_FORGE_EMSCRIPTEN_ACTIVATED+x} ]; then
+
+
+
     export CONDA_FORGE_EMSCRIPTEN_ACTIVATED=1
 
     export EMSDK_PYTHON=${BUILD_PREFIX}/bin/python3
     export PYTHON=${BUILD_PREFIX}/bin/python3
 
     emsdk install  3.1.2
```

Comment:
How about checking for another env var here? Such as `CONDA_EXTERNAL_EMSDK` or so, and if that one is set, use that EMSDK? So that we can keep both functionalities.

Reply:
This is important if we ever want to move to conda-forge...

Reply:
yeah, was thinking about the same.

But this pr is basically just trying to fix the "broken" state that each local build forces a download of the emscripten toolchain ... I consider this as a bug/broken